CTFd/themes/admin/assets/js/pages/configs.js

```diff
@@ -374,6 +412,7 @@ $(() => {
   insertTimezones($("#freeze-timezone"));
 
   $(".config-section > form:not(.form-upload)").submit(updateConfigs);
+  $("#user-mode").change(switchMode);
```

Comment:
This is the wrong selector.

Reply:
You also have the additional problem here where if the user says no, the change is still loaded in the UI. So if a user starts a change, selects no, and then hits save, the change will still be made. 

I feel like you may want to consider an alternative approach. 

Some options off the top of my head:

- Don't automatically try to reset submissions and just tell the user that they should reset it themselves afterwards. I kind of think this may be the best because it doesn't cause data loss. As long as the user is aware of what's going on, I think it's overall okay.  
- Add another checkbox in the reset flow specifically for user_mode reset
- Handle resetting in the configs API route.

Reply:
That is true! Would a no function call in the ezQuery function be out of the question? IE the code below in ezq.js
```javascript
$(no).click(function() {
    args.reject();
  });
```
This would allow for setting the UI value back to the previous value if a user says no. This is because there may have to be a specific endpoint for changing from teams_mode to user_mode because there has to be some data loss. Teams data should be deleted and Users should have no team_id at the minimum and currently there is only the accounts deletion in the reset endpoint which deletes users and teams.